src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/util/ManagedIndexUtils.kt

```diff
@@ -398,6 +398,15 @@ val ManagedIndexMetaData.isFailed: Boolean
         return false
     }
 
+val ManagedIndexMetaData.isPolicyCompleted: Boolean
+    get() {
+        // If policyCompleted is true, return true
+        if (this.policyCompleted == true) return true
```

Comment:
this can be simplified to 

```
val ManagedIndexMetaData.isPolicyCompleted: Boolean
    get() = this.policyCompleted == true
```

also why do we need as separate variable? Can we achieve the same thing by just doing below?

```
if (policy.policyCompleted == true) {
    ...
}
```

Reply:
True but I didn't like mixing predicate style ```isFailed``` with explicit boolean evaluation like ```policyCompleted == true```, it made the boolean statement less readable and inconsistent.